lib/invoice_item.rb

```diff
@@ -0,0 +1,23 @@
+class InvoiceItem
+  attr_reader :id,
+              :item_id,
+              :invoice_id,
+              :quantity,
+              :unit_price,
+              :created_at,
+              :updated_at
+
+  def initialize(details)
+    @id = details[:id]
+    @item_id = details[:item_id]
+    @invoice_id = details[:invoice_id]
+    @quantity = details[:quantity]
+    @unit_price = details[:unit_price]
```

Comment:
⛏ You should check the spec harness to see if they want the price returned as a BigDecimal object! You should be able to read the iteration 3 tests and see what they need there.

Reply:
The ip specifies to initialize with a Bigdecimal there, but I'll adjust it to the spec harness. I'll wait till we merge in your spec harness work then adjust this to comply.